src/code-snippet/code-snippet.component.ts

```diff
@@ -231,7 +231,7 @@ export class CodeSnippet extends BaseIconButton implements OnInit, AfterViewInit
 		let ref;
 		switch (this.display) {
 			case "multi":
-				ref = this.codeContent.nativeElement;
+				ref = this.codeContent?.nativeElement;
```

Comment:
```suggestion
				ref = this.codeContent.nativeElement;
```

Same as comment below. This _may_ be required.